src/index.js

```diff
@@ -71,27 +107,40 @@ async function fetchData(user) {
   )
 
   let bills = []
+
   for (const beneficiary of beneficiaries) {
     const name = beneficiary.insurance_profile.user.normalized_full_name
+
     bills.push.apply(
       bills,
-      beneficiary.insurance_profile.settlements
-        .filter(bill => bill.reimbursement_status === 'processed')
+      events
+        .filter(bill => bill.status === 'refunded')
         .map(bill => ({
           vendor: 'alan',
-          vendorRef: bill.created_at,
+          vendorRef: bill.id,
           beneficiary: name,
           type: 'health_costs',
           date: moment(bill.estimated_payment_date, 'YYYY-MM-DD').toDate(),
           originalDate: moment(bill.care_date, 'YYYY-MM-DD').toDate(),
-          subtype: bill.displayed_label,
-          description: bill.care_type_desc,
-          socialSecurityRefund: bill.ss_amount / 100,
-          amount: bill.covered_amount / 100,
-          originalAmount: bill.total_amount / 100,
-          isThirdPartyPayer: bill.origin === 'tiers_payant',
+          subtype: bill.care_acts[0].display_label,
+          socialSecurityBase: bill.care_acts[0].ss_base / 100,
```

Comment:
Ce champ doit rester socialSecurityRefund comme dans : https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.bills.md#optional-attributes-but-some-are-important-depending-the-context